src/data.js

```diff
@@ -141,39 +141,45 @@ Namespace.ensureNamespace = function(namespace, expires) {
 };
 
 /**Delete expired entries */
-Namespace.expireEntries = function(parent, indexedTask, continuationToken=null) {
-  console.log(`expireEntries in '${parent}' with token '${continuationToken}'`);
-  return this.query({
-    parent: parent,
+Namespace.expireEntries = function(indexedTask, continuationToken=null) {
+  //console.log(`expireEntries in '${parent}' with token '${continuationToken}'`);
+  return this.scan({
```

Comment:
Azure can do the filtering for expired entries for you:

https://github.com/taskcluster/taskcluster-queue/blob/a2007b195c0cd875b64ea29372731ccd5e760fc5/src/data.js#L301-L303

which will save downloading all those non-expired entities.  Which (after the first run) will be most of them!